internal/util/path_test.go

```diff
@@ -14,7 +14,7 @@ func TestGetApplicationDir(t *testing.T) {
 
 	dir, err := GetApplicationDir()
 	a.Nil(err, "GetApplicationDir() failed with error  %v", err)
-	a.Equal(true, strings.HasSuffix(dir, ".twitch-cli"), "GetApplicationDir() expected to end with %v, got %v", ".twitch-cli", dir)
+	a.Equal(true, strings.HasSuffix(dir, "twitch-cli"), "GetApplicationDir() expected to end with %v, got %v", ".twitch-cli", dir)
```

Comment:
the expected `%v` argument still has a dot.

Reply:
Heh, caught that. :)